### ISSUE TITLE
Received invalid command: diagnostics/get

### ISSUE THREAD

**Issue by calisro** [home-assistant/frontend#11517]
### The problem

After upgrading to 2022.2 I am seeing this error on startup.  It may be happening more often but not sure yet.

```
Logger: homeassistant.components.websocket_api.http.connection
Source: components/websocket_api/connection.py:96
Integration: Home Assistant WebSocket API ([documentation](https://www.home-assistant.io/integrations/websocket_api), [issues](https://github.com/home-assistant/home-assistant/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+websocket_api%22))
First occurred: 7:05:45 PM (2 occurrences)
Last logged: 7:05:50 PM

[548054276416] Received invalid command: diagnostics/get
```

### What version of Home Assistant Core has the issue?

core-2022.2.0

### What was the last working version of Home Assistant Core?

core-2021.x

### What type of installation are you running?

Home Assistant OS

**Comment by balloob** [comment]
We need to guard if user has diagnostics set up before fetching data.

**Comment by calisro** [comment]
IS there something the user needs to setup?  I assume I'm missing a config include?

**Comment by balloob** [comment]
Well you don't use `default_config` and so don't have `diagnostics` loaded.

**Comment by calisro** [comment]
Yes I just added that today.  That wasn't listed in any documentation nor release notes.

**Comment by bramkragten** [comment]
That is what he says:

> We need to guard if user has diagnostics set up before fetching data.

We shouldn't fetch diagnostics data if the user has not set up diagnostics.
And that is fixed now in #11518 :-)
